examples/react-instantsearch/src/Autocomplete.js

```diff
@@ -0,0 +1,28 @@
+import { autocomplete } from '@algolia/autocomplete-js';
+import React, { createElement, Fragment, useEffect, useRef } from 'react';
```

Comment:
this file doesn't have the jsx pragma at the top, is it needed?

Reply:
For some reason it's needed with this Parcel configuration with React. This file doesn't require it though because we don't use JSX.

Reply:
why does the other file require it?

Reply:
The files that require it are the ones using JSX, because Babel need to know what pragma to use to transform JSX to `createElement` calls.

Reply:
but this file also uses jsx 🤔

Reply:
Isn't it because `createElement` is in scope of this file?

Reply:
Ah true, @shortcuts is right!